### DISCUSSION TITLE
External Drum pads with hydrogen

### DISCUSSION THREAD

**Discussion by zensky1101** [hydrogen-music/hydrogen#1165] [Hydrogen on Linux]
Hello does anyone know if hydrogen will work with external drum hardware like a Roland octapad. Or any external hardware for that matter?

**Comment by oddtime** [comment]
Have you tried with midi? I have done it with yamaha electronic drums, via jack, and it worked

**Reply by MythicMike** [reply]
Hello,

Works with eDrum via midi output. In my config, I have combined 2 different edrums:
- 1 kit, XM 100s (seems to be only known in Taiwan and I am certainly one of the rare owner of such edrum in France) connected via midi (out --> in) to 1 kit, Roland TDV6 and then connected via midi to PC USB (out --> PC USB).
This works, just need to select the midi input and to set "ignore midi off messages" in the Midi preferences.

Hoping, it helps.

Brs

Michel
